### PR TITLE
DG updates for manual testing

### PR DESCRIPTION
**Pull request by ivantjh**
- Add skeleton for manual testing

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -346,60 +346,38 @@ _{Explain here how the data archiving feature will be implemented}_
...
 |                     |                    |                                                                                        |                                                                     |
-| Profile             | Forgetful User     | Add a profile picture to the contact                                                   | I can remember how they look like                                   |
+| Relationship management   | Good friend        | Set a desired frequency (weekly. monthly, yearly) for meeting each contact       | I can set a goal on how frequent to meet my friends.                |
+| Relationship management   | Forgetful friend   | Be reminded of who i should meet soon                                            | I do not forget to meet my friends.                                 |
+| Relationship management   | Friend             | See upcoming dates (birthdays/ special dates)                                    | I can prepare myself before I meet them/                            |
```
Comment (chesterhow) [inline docs/DeveloperGuide.md]: Can remove the `/` at the end of the line

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -553,7 +532,61 @@ testers are expected to do more *exploratory* testing.
...
-### Deleting a person
+### Saving data
+
+1. Dealing with missing/corrupted data files
+
+    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+
+1. _{ more test cases …​ }_
+
+### Add a person: `add`
```
Comment (chesterhow) [inline docs/DeveloperGuide.md]: Can we change this to `Adding a person` to be more consistent with the rest?

**Review by chesterhow — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -346,60 +346,38 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| Friendship Journal  | User               | Record brief summary of last meeting with the other person (include last meeting date) | I know why we last met.                                             |
 |                     |                    |                                                                                        |                                                                     |
-| Storage/backup/data | Tech-Savvy User    | Export/import information from the application                                         | I can back up the data                                              |
-| Storage/backup/data | Tech-Savvy User    | Detect conflicts when importing information                                            | I can detect if there are any changes to the after exporting        |
+| UX                  | Forgetful User     | See suggested commands as i type in the command bar                                    | I do not have to memorise all commands                              |
```
Comment (lue97) [inline docs/DeveloperGuide.md]: capital 'i'

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -346,60 +346,38 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| Search              | Experienced User   | Write SQL like statements to search for things                                         | I can have full control over search                                 |
 |                     |                    |                                                                                        |                                                                     |
-| Profile             | Forgetful User     | Add a profile picture to the contact                                                   | I can remember how they look like                                   |
+| Relationship management   | Good friend        | Set a desired frequency (weekly. monthly, yearly) for meeting each contact       | I can set a goal on how frequent to meet my friends.                |
+| Relationship management   | Forgetful friend   | Be reminded of who i should meet soon                                            | I do not forget to meet my friends.                                 |
```
Comment (lue97) [inline docs/DeveloperGuide.md]: capital 'i'

**Review by lue97 — CHANGES_REQUESTED** [review]
minor typos

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -569,12 +602,32 @@ testers are expected to do more *exploratory* testing.
...
+
+### Locating persons by name: `find`
+
+### Viewing help: `help`
+
+### Listing all persons: `list`
+
+### Setting meeting goal: `set-goal`
+
+### Subtracting Debt: `subract-debt`
```
Comment (lue97) [inline docs/DeveloperGuide.md]: typo in 'subtract'

**Review by lue97 — APPROVED** [review]
LGTM

**Pull request merged**